code/modules/awaymissions/maploader/reader.dm

```diff
@@ -321,7 +325,7 @@ var/global/dmm_suite/preloader/_preloader = null
 /dmm_suite/preloader/New(list/the_attributes, path)
 	.=..()
 	if(!the_attributes.len)
-		Del()
+		qdel(src)

```

Comment:
The ```qdel``` proc correctly process QDEL_HINT_HARDDEL_NOW even if the garbage subsystem isn't operational.

What i don't get is why adding the overhead of a function call and several checks to a proc generally only called once (at server startup) and quite time consumming already ?